rllib/algorithms/dreamer/dreamer_model.py

```diff
@@ -550,6 +550,8 @@ def get_initial_state(self) -> List[TensorType]:
         self.state = self.dynamics.get_initial_state(1) + [
             torch.zeros(1, self.action_space.shape[0]).to(self.device)
         ]
+        # returned state should be of shape (state_dim, )
```

Comment:
This bug was discovered when I compared the results of the new and old approach.